lib/ember-test-helpers/test-module-for-component.js

```diff
@@ -190,6 +190,70 @@ export default TestModule.extend({
     };
   },
 
+  setupComponentLegacyIntegrationTest: function () {
+    var resolver = getResolver();
+    var namespace = Ember.Object.create({
+      Resolver: { create: function() { return resolver; } }
+    });
+
+    if (Ember.Application.buildRegistry) {
+      var registry;
+      registry = Ember.Application.buildRegistry(namespace);
+      registry.register('component-lookup:main', Ember.ComponentLookup);
+      this.registry = registry;
+      this.container = registry.container();
+    } else {
+      this.container = Ember.Application.buildContainer(namespace);
+      this.container.register('component-lookup:main', Ember.ComponentLookup);
+    }
+  },
+
+  setupLegacyComponentTest: function () {
```

Comment:
How does this differ from https://github.com/switchfly/ember-test-helpers/blob/master/lib/ember-test-helpers/test-module-for-component.js#L72-L120?  It seems that these are essentially duplicated...